R/mod_chart_example.R

```diff
@@ -40,24 +38,24 @@ mod_chart_example_server <- function(id) {
     ns <- session$ns
 
     output$chart <- highcharter::renderHighchart({
-
+      
       # Generate bins based on input$bins from ui.R
       x <- nhsbsaShinyR::faithful[, 2]
       bins <- seq(min(x), max(x), length.out = as.numeric(input$bins) + 1)
 
       # Draw the histogram with the specified number of bins
-      chart <- hist(x, breaks = bins)
+      chart <- graphics::hist(x, breaks = bins, plot = FALSE)
 
       # Output interactive chart
       chart %>%
         highcharter::hchart() %>%
-        theme_nhsbsa()
+        nhsbsaR::theme_nhsbsa_highchart()
```

Comment:
Use the exported function from `nhsbsaR`, rather than defining it in this package. Also, it was an old version defined in this package so better to get the newest version.